package.json

```diff
@@ -68,6 +68,7 @@
     "superagent-proxy": "^1.0.2"
   },
   "devDependencies": {
+    "assert": "^1.4.1",
```

Comment:
There's no need to introduce this dependency as node.js already has it: https://nodejs.org/dist/latest-v8.x/docs/api/assert.html. In fact, this dependency is derived from the node.js assert module.